play/dbt-materialize/.gitignore

```diff
@@ -0,0 +1,78 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Is this .gitignore file copied from somewhere else / recommended by dbt / other? I prefer to see .gitignore files that are as minimal as possible / only include files that cannot be generated / placed somewhere else.

Reply:
It seems like I can just get rid of this `.gitignore`!